downstream/assemblies/platform/assembly-configure-egress-proxy.adoc

```diff
@@ -0,0 +1,24 @@
+[id="assembly-configure-egress-proxy"]
+
+= Configuring Ansible Automation Platform to use egress proxy
+
+You can deploy {PlatformNameShort} so that egress from the platform for various purposes functions properly through proxy servers. 
+Egress proxy allows clients to make indirect (through a proxy server) requests to network services. 
+The client first connects to the proxy server and requests some resource, for example, email, located on another server. 
+The proxy server then connects to the specified server and retrieves the resource from it.
+
+
+== Overview
+To egress proxy should be configured on the system and component level of {PlatformNameShort}, for all the installation methods (RPM and containerized). 
```

Comment:
Change "To egress proxy..." to "The egress proxy..."

Also IIUC, this is not true for OCP installations? If so, this statement might be misleading. I suggest changing to:
 
The egress proxy should be configured on the system and component level of {PlatformNameShort}, for the RPM and containerized installation methods.